content/docs/models/crud_operations.md

```diff
@@ -97,6 +97,18 @@ const user = await User.findBy('email', 'virk@adonisjs.com')
 // SQL: SELECT * from "users" WHERE "email" = 'virk@adonisjs.com' LIMIT 1;
 ```
 
+### findManyBy
+
+Find multiple record by one or multiple column name and its value. This method returns an array of model instance or `null`.
```

Comment:
Should it return `null` or an empty array? Because all other queries returns empty array  when the successful result is an array